linux/src/async_socket_linux.c

```diff
@@ -298,6 +298,8 @@ static void internal_close(ASYNC_SOCKET_HANDLE async_socket)
         LogError("Failure removing socket from epoll_ctrl");
     }
 
+    LogInfo("async socket is closing socket: %" PRI_MU_SOCKET "", async_socket->socket_handle);
```

Comment:
>PRI_MU_SOCKET [](http://example.com/codeflow?start=48&length=13)

This should be renamed please.
MU stands for macro utils. So this, not having anything to do with macro utils should be named PRI_SOCKET

Reply:
I noticed this when I picked it up last time in zrpc, but did not have the time to go back and fix it. But now that you are changing anyway c-pal and will have to propagate it makes sense to fix it.